src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -179,10 +179,22 @@ static u8 D_80B65078[] = {
 
 // used in limbdraw
 static Vec3f D_80B65084[] = {
-    { 2000.0f, 4000.0f, 0.0f,},
-    {-1000.0f, 1500.0f, -2000.0f,},
-    {-1000.0f, 1500.0f, 2000.0f,},
-}; 
+    {
+        2000.0f,
+        4000.0f,
+        0.0f,
+    },
+    {
+        -1000.0f,
+        1500.0f,
+        -2000.0f,
+    },
+    {
+        -1000.0f,
+        1500.0f,
+        2000.0f,
+    },
```

Comment:
Lose all the commas on the inner arrays here and reformat: we don't want it like this.

Reply:
Can you elaborate further? I can't seem to delete any commas here without breaking the build

Reply:
Oh I think I understand. You mean delete the final commas of the inner arrays, right?

Reply:
Yes, those should all end up on one line. Test with or without the final comma on the outer one too, one may look better.